projects/demo/src/modules/components/button/examples/4/index.less

```diff
@@ -1,15 +1,16 @@
+@import 'taiga-ui-local';
 // This goes into global, unencapsulated styles
 [tuiWrapper][data-appearance='custom'] {
     background: #bc71c9;
     color: #fff;
 
-    &[data-state='hovered'] {
+    .wrapper-hover({
         background: #a381ff;
-    }
+    });
 
-    &[data-state='pressed'] {
+    .wrapper-active({
         background: #8f75d1;
-    }
+    });
 
     &[data-state='disabled'] {
```

Comment:
I believe there's a mixin for that as well. Better to use those so it's centralized.